platform/machine/do/machine.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package do
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/digitalocean/godo"
+	"golang.org/x/crypto/ssh"
+
+	"github.com/coreos/mantle/platform"
+)
+
+type machine struct {
+	cluster   *cluster
+	droplet   *godo.Droplet
+	journal   *platform.Journal
+	publicIP  string
+	privateIP string
+}
+
+func (pm *machine) ID() string {
+	return strconv.Itoa(pm.droplet.ID)
+}
+
+func (pm *machine) IP() string {
+	return pm.publicIP
+}
+
+func (pm *machine) PrivateIP() string {
+	return pm.privateIP
+}
+
+func (pm *machine) SSHClient() (*ssh.Client, error) {
+	return pm.cluster.SSHClient(pm.IP())
+}
+
+func (pm *machine) PasswordSSHClient(user string, password string) (*ssh.Client, error) {
+	return pm.cluster.PasswordSSHClient(pm.IP(), user, password)
+}
+
+func (pm *machine) SSH(cmd string) ([]byte, []byte, error) {
+	return pm.cluster.SSH(pm, cmd)
+}
+
+func (m *machine) Reboot() error {
```

Comment:
I noticed this pattern elsewhere but figured I'd ask here. Is there a reason that the `Reboot` function names the `*machine` variable as `m` rather than the name being used in all of the others (`pm` in this instance)?

Reply:
Probably it happened once and then was copypasted everywhere.  I'll add a patch to fix up the other instances.